### PR TITLE
Bundle/theming

### PR DESCRIPTION
**Pull request by fraincs**
Issue:
[308](https://github.com/gsoft-inc/sg-orbit/issues/308) / [304](https://github.com/gsoft-inc/sg-orbit/issues/304) / [306](https://github.com/gsoft-inc/sg-orbit/issues/306) / [307](https://github.com/gsoft-inc/sg-orbit/issues/307)

### CODE REVIEW

File: packages/semantic-ui-theme/semantic/src/themes/sharegate/elements/input.overrides [hunk 1]
```diff
@@ -124,10 +124,16 @@
     padding-right: calc(var(--scale-echo) + var(--scale-golf));
 }
 
-.with-button .ui.input + .input-clear-button {
+.with-button .ui.input + .input-clear-button, {
     right: calc(var(--scale-echo) / 2);
 }
 
+/* Date Picker */
```
Comment (patricklafrance) [inline packages/semantic-ui-theme/semantic/src/themes/sharegate/elements/input.overrides:131]: I don't think an input should be aware that a date picker is using an input

File: packages/semantic-ui-theme/semantic/src/themes/sharegate/elements/label.overrides [hunk 2]
```diff
@@ -493,6 +492,110 @@
     padding-left: calc(var(--scale-foxtrot) / 2 - 4px);
 }
 
+/* Default - With Avatar  */
```
Comment (patricklafrance) [inline packages/semantic-ui-theme/semantic/src/themes/sharegate/elements/label.overrides:495]: I don't think a label should be aware that a dropdown is use it

File: packages/react-components/src/select/src/select.jsx [hunk 3]
```diff
@@ -113,7 +113,11 @@ export function PureSelect(props) {
                 </>
             );
 
-            additionalClasses = "with-icons";
+            additionalClasses = mergeClasses(
+                "with-icons",
+                !isNil(right) && right && "with-icons-right",
+                !isNil(left) && left && "with-icons-left",
```
Comment (patricklafrance) [inline packages/react-components/src/select/src/select.jsx:119]: you should use `iconsPosition === "right"` instead.

```
            additionalClasses = mergeClasses(
                "with-icons",
                iconsPosition === "right" ? "with-icons-right" : "with-icons-left"
```

**Comment by patricklafrance** [comment]
There seems to be a bug with a radio having a label, the label is very small:

image

**Comment by patricklafrance** [comment]
This PR should also make sure overflow works fine for Toggles

**Comment by patricklafrance** [comment]
I added docs and chromatic tests for Radio button Count support.

**Comment by patricklafrance** [comment]
Seems like A LOT of padding between the icon and the text?

image

**Comment by patricklafrance** [comment]
Also seem like a HUGE indent?

image

**Comment by fraincs** [comment]
> Seems like A LOT of padding between the icon and the text?
> 
> image

C'est ce qui a été établi. Soit insérer [16 | 16 ] l'icone dans le milieu du padding.

**Comment by patricklafrance** [comment]
MultiSelect small label is too big for his button:

image

**Pull request merged**